zkstack_cli/crates/zkstack/src/commands/ecosystem/args/init.rs

```diff
@@ -13,13 +14,20 @@ use crate::{
     commands::chain::args::{genesis::GenesisArgs, init::da_configs::ValidiumTypeArgs},
     defaults::LOCAL_RPC_URL,
     messages::{
-        MSG_DEPLOY_ECOSYSTEM_PROMPT, MSG_DEPLOY_ERC20_PROMPT, MSG_DEV_ARG_HELP,
+        MSG_BRIDGEHUB, MSG_DEPLOY_ECOSYSTEM_PROMPT, MSG_DEPLOY_ERC20_PROMPT, MSG_DEV_ARG_HELP,
         MSG_L1_RPC_URL_HELP, MSG_L1_RPC_URL_INVALID_ERR, MSG_NO_PORT_REALLOCATION_HELP,
         MSG_OBSERVABILITY_HELP, MSG_OBSERVABILITY_PROMPT, MSG_RPC_URL_PROMPT,
         MSG_SERVER_COMMAND_HELP, MSG_SERVER_DB_NAME_HELP, MSG_SERVER_DB_URL_HELP,
     },
 };
 
+#[derive(Clone, Copy, Default)]
+pub struct PromptPolicy {
+    pub deploy_erc20: bool,
+    pub observability: bool,
+    pub skip_ecosystem: bool,
+}
```

Comment:
what does it mean?

Reply:
this means that we can choose which prompts to display to the user instead of copy/pasting the code each time